src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/repository/PersonRepositoryIntTest.kt

```diff
@@ -1,42 +1,49 @@
 package uk.gov.justice.digital.hmpps.personrecord.jpa.repository
 
 import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
-import org.springframework.test.context.jdbc.Sql
-import org.springframework.test.context.jdbc.SqlConfig
 import uk.gov.justice.digital.hmpps.personrecord.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.personrecord.jpa.entity.DefendantEntity
+import uk.gov.justice.digital.hmpps.personrecord.jpa.entity.OffenderEntity
+import uk.gov.justice.digital.hmpps.personrecord.jpa.entity.PersonEntity
+import uk.gov.justice.digital.hmpps.personrecord.model.OtherIdentifiers
+import uk.gov.justice.digital.hmpps.personrecord.model.Person
 import uk.gov.justice.digital.hmpps.personrecord.validate.PNCIdentifier
 import java.time.LocalDate
 import java.util.*
 
-@Sql(
-  scripts = ["classpath:sql/before-test.sql"],
-  config = SqlConfig(transactionMode = SqlConfig.TransactionMode.ISOLATED),
-  executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD,
-)
-@Sql(
-  scripts = ["classpath:sql/after-test.sql"],
-  config = SqlConfig(transactionMode = SqlConfig.TransactionMode.ISOLATED),
-  executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD,
-)
 class PersonRepositoryIntTest : IntegrationTestBase() {
 
+  @BeforeEach
+  fun before() {
+    personRepository.deleteAll()
+  }
+
   @Test
   fun `should return exact match for provided UUID`() {
     // Given
     val personId = UUID.fromString("eed4a9a4-d853-11ed-afa1-0242ac120002")
+    val person = Person(otherIdentifiers = OtherIdentifiers(pncIdentifier = PNCIdentifier("2001/0171310W"), crn = "CRN1234"), givenName = "Iestyn", familyName = "Mahoney", dateOfBirth = LocalDate.of(1965, 6, 18))
+
+    val newPersonEntity = PersonEntity(personId = personId)
+    newPersonEntity.createdBy = "test"
+    newPersonEntity.lastUpdatedBy = "test"
+    val newDefendantEntity = DefendantEntity.from(person)
+    newDefendantEntity.person = newPersonEntity
+    newPersonEntity.defendants.add(newDefendantEntity)
+    personRepository.saveAndFlush(newPersonEntity)
 
     // When
     val personEntity = personRepository.findByPersonId(personId)
 
     // Then
-    assertThat(personEntity).isNotNull
-    assertThat(personEntity?.defendants).hasSize(2)
+    assertThat(personEntity?.defendants).hasSize(1)
     val hmctsDefendantEntity = personEntity?.defendants?.get(0)
     assertThat(hmctsDefendantEntity?.pncNumber).isEqualTo(PNCIdentifier("2001/0171310W"))
     assertThat(hmctsDefendantEntity?.crn).isEqualTo("CRN1234")
     assertThat(hmctsDefendantEntity?.forenameOne).isEqualTo("Iestyn")
-    assertThat(hmctsDefendantEntity?.forenameTwo).isEqualTo("Carey")
```

Comment:
The `from` method does not allow you to add these fields at the moment so I have removed this

Reply:
Probably a hangover from many iterations. Will tidy up in the next sprint.